### PR TITLE
various into/odo name changes

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

File: odo/backends/h5py.py [hunk 1]
```diff
@@ -161,9 +161,9 @@ def create_from_datashape(group, ds, name=None, **kwargs):
             "Trying to create an HDF5 file with non-record datashape failed\n"
             "Perhaps you forgot to specify a datapath?\n"
             "\tdshape: %s\n"
-            "If you're using into consider the following change\n"
-            "\tBefore: into('myfile.hdf5', data)\n"
-            "\tAfter:  into('myfile.hdf5::/datapath', data)" % ds)
+            "If you're using odoconsider the following change\n"
```
Comment (cpcloud) [inline odo/backends/h5py.py]: space here

File: odo/backends/hdfstore.py [hunk 2]
```diff
@@ -55,7 +55,7 @@ def hdfstore_to_chunks_dataframes(data, **kwargs):
 
     import tables
 
-before you import projects like into or blaze."""
+before you import projects like odo or into or blaze."""
```
Comment (cpcloud) [inline odo/backends/hdfstore.py]: should this be `odo or blaze`?

Comment (mrocklin) [inline odo/backends/hdfstore.py]: Figured the error message could lag a bit

Comment (cpcloud) [inline odo/backends/hdfstore.py]: cool

**Pull request merged**